torch_geometric/nn/models/basic_gnn.py

```diff
@@ -192,6 +192,50 @@ def MLP(in_channels: int, out_channels: int) -> torch.nn.Module:
         )
 
 
+class GINE(BasicGNN):
```

Comment:
We can also inherit from `GIN`, which makes the `GIN.MLP` call a little bit more cleaner.